content/blog/making-a-slack-bot-with-node-and-mongo/index.md

```diff
@@ -281,7 +281,7 @@ console.log(state); // {word1: 2, word2: -1}
 
 Following this pattern, let's go through and add a few lines of code to the last example to fulfill the expected behavior:
 
-```javascript
+````javascript
```

Comment:
Actually, scratch that - I've found what causes this. There are three backticks used in part of this code snippet (````text: '```\n' + tableString + '```',````), so it should be wrapped with four backticks to format it correctly. Our markdown renderer still handles this fine because the backticks in question are enclosed by other characters, but this still makes sense as a linting rule.